libs/designer/src/lib/core/actions/bjsworkflow/serializer.ts

```diff
@@ -59,13 +60,35 @@ export interface SerializeOptions {
 
 export const serializeWorkflow = async (rootState: RootState, options?: SerializeOptions): Promise<Workflow> => {
   if (!options?.skipValidation) {
+    const intl = getIntl();
+
+    const operationsWithConnectionErrors = Object.entries(rootState.operations.errors).filter(
+      ([_id, errors]) => !!errors[ErrorLevel.Connection]
+    );
+    if (operationsWithConnectionErrors.length > 0) {
+      throw new Error(
+        intl.formatMessage(
```

Comment:
is it possible to show a intl message to the use but throw a english error message. This is going to make it hard to look for in logs.

Reply:
I thought this message would be visible in UI, but yes we can throw an exception which will have innerException details which i guess should be logged by Portal right?